packages/loot-core/src/server/util/rschedule.ts

```diff
@@ -1,5 +1,4 @@
 import '@rschedule/standard-date-adapter/setup';
-import '@rschedule/json-tools/Schedule';
```

Comment:
This import add's the functions `toJSON` and `fromJSON` and removes all other types. These functions are not used in the cureent code base.